shared/src/components/Product/productCopies.tsx

```diff
@@ -501,6 +501,62 @@ export const productCopies: { [vault in VaultOptions]: ProductCopies } = {
       ),
     },
   },
+  "rsAVAX-THETA": {
+    title: "T-sAVAX-C",
+    subtitle: "sAVAX Call",
+    description:
+      "Generates yield by running an automated sAVAX covered call strategy.",
+    tags: ["COVERED CALL"],
+    strategy: (
+      <>
+        The vault earns yield on its sAVAX deposits by running a weekly
+        automated sAVAX{" "}
+        <TooltipExplanation
+          title="COVERED CALL"
+          explanation="A covered call refers to a financial transaction in which the investor selling call options owns an equivalent amount of the underlying security."
+          learnMoreURL="https://www.investopedia.com/terms/c/coveredcall.asp"
+          renderContent={({ ref, ...triggerHandler }) => (
+            <HighlightedText ref={ref} {...triggerHandler}>
+              covered call
+            </HighlightedText>
+          )}
+        />{" "}
+        strategy. The vault reinvests the yield earned back into the strategy,
+        effectively compounding the yields for depositors over time.
```

Comment:
Here we can add the bit about depositing into BENQI's staked avax contract, like with Lido

https://github.com/ribbon-finance/ribbon-frontend/blob/26855b78f56594f55587b2b5e9a251f0bf6561c6/shared/src/components/Product/productCopies.tsx#L332-L353

```js
 <>
        The vault earns yield on its ETH deposits by running a weekly automated
        ETH{" "}
        <TooltipExplanation
          title="COVERED CALL"
          explanation="A covered call refers to a financial transaction in which the investor selling call options owns an equivalent amount of the underlying security."
          learnMoreURL="https://www.investopedia.com/terms/c/coveredcall.asp"
          renderContent={({ ref, ...triggerHandler }) => (
            <HighlightedText ref={ref} {...triggerHandler}>
              covered call
            </HighlightedText>
          )}
        />{" "}
        strategy where it stakes its ETH deposits in{" "}
        <TooltipExplanation
          title="LIDO"
          explanation="Lido empowers stakers to put their staked assets to use. "
          learnMoreURL="https://lido.fi"
          renderContent={({ ref, ...triggerHandler }) => (
            <HighlightedText ref={ref} {...triggerHandler}>
              Lido
            </HighlightedText>
          )}
        />{" "}
        and then uses its{" "}
        <TooltipExplanation
          title="STETH"
          explanation="stETH is the deposit token that represents a user's share of the their ETH on the Ethereum beacon chain."
          learnMoreURL="https://lido.fi/ethereum"
          renderContent={({ ref, ...triggerHandler }) => (
            <HighlightedText ref={ref} {...triggerHandler}>
              stETH
            </HighlightedText>
          )}
        />{" "}
        to collateralize weekly out-of-money ETH call options. The yield earned
        from both the covered call strategy and the ETH staking rewards are
        reinvested weekly, effectively compounding the yields for depositors
        over time.
      </>
```

Reply:
Done